kpm.go

```diff
@@ -37,6 +37,7 @@ func main() {
 		cmd.NewLogoutCmd(kpmcli),
 		cmd.NewPushCmd(kpmcli),
 		cmd.NewPullCmd(kpmcli),
+		cmd.NewUpdateCmd(kpmcli),
```

Comment:
Need to synchronize CLI implementation to the cli repository and update the kpm dependency to the latest main branch like the `kcl mod push` command https://github.com/kcl-lang/cli/blob/main/cmd/kcl/commands/mod_push.go

Reply:
OK